pkg/proxy/ipvs/proxier.go

```diff
@@ -1295,11 +1295,11 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 
 		// Capture load-balancer ingress.
-		for _, ingress := range svcInfo.LoadBalancerIPStrings() {
-			if ingress != "" {
+		for _, ingress := range svcInfo.LoadBalancerIngress() {
+			if ingress.IP != "" && ingress.RouteType != v1.LoadBalancerRouteTypeProxy {
```

Comment:
if kube-proxy runs against a previous version of the APIserver this field would not be available, should we check first if the field exists?

Reply:
🤔 in this case the field would just be an empty string no?

Reply:
In general we say that clients can be older than servers but not newer.  If you want to be pedantic, you check for nil and handle it as the default

Reply:
@thockin do you rather see a IPMode as a `string` or as a `*string`. If we set a default on create, I think we can safely use a `string` and check of empty string instead of `nil`?